src/style.css

```diff
@@ -0,0 +1,29 @@
+html,
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--1 .stars__star:nth-child(-n + 1),
```

Comment:
hint: in this case you also can use direct number or just `:first-child`
for the symmetry it looks cool but every formula with `n` adds some small but additional calculation to you page

Reply:
What about :last-child, it can be used to?

Reply:
So, i fixed it and remove last pseudo-element to. Now it looks like just '.stars--1 .stars__star'.